app/helpers/application_helper/button/mass_transform_vm_button.rb

```diff
@@ -0,0 +1,15 @@
+class ApplicationHelper::Button::MassTransformVmButton < ApplicationHelper::Button::Basic
+  def visible?
+    true
+  end
+
+  def disabled?
+    @error_message = _('No suitable target infra provider exists') unless destination_exists?
+    @error_message.present?
+  end
+
+  def destination_exists?
+    # Is there a provider that supports import?
+    !ManageIQ::Providers::Redhat::InfraManager.all.select(&:validate_import_vm).empty?
```

Comment:
Prefer using detect as it will stop on first successful condition check

Reply:
@ohadlevy What does it return? I cannot find it anywhere.

Reply:
@ohadlevy  Done.